File: 1-Iniciante/JavaScript/1020.js
let qtdDias = parseInt(input)

let qtdAnos, qtdMeses

qtdAnos = parseInt(qtdDias / 365)
qtdDias = qtdDias % 365

qtdMeses = parseInt(qtdDias / 30)
qtdDias = qtdDias % 30

let resultado = `${qtdAnos} ano(s)
${qtdMeses} mes(es)
${qtdDias} dia(s)`
console.log(resultado)

File: 1-Iniciante/Java/1020.java
import java.io.IOException;
import java.util.Scanner;
 
public class Main {
 
    public static void main(String[] args) throws IOException {
 
       Scanner in = new Scanner(System.in);
        int valor = in.nextInt();
        int anos = valor / 365;
        int mes = (valor % 365)/30;
        int dias = valor - (anos*365) - (mes*30);
        
        System.out.printf("%d ano(s)\n", anos);
        System.out.printf("%d mes(es)\n", mes);
        System.out.printf("%d dia(s)\n", dias);
        
    }
 
}
